translations/translation_cz.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8" standalone="no" ?>
+<l10n>
+    <translationContributors>Bodzio528</translationContributors>
```

Comment:
Hi, please add yourself as translation contributor, take some credit for your hard work :)